### PR TITLE
[Related #93] Player list template

### PR DESCRIPTION
**Pull request by supunTE**
- created playerlist template (component) and its story
- created room page (there're some updates from another PR)

### CODE REVIEW

File: apps/client/src/pages/room/Room.page.tsx [hunk 1]
```diff
@@ -1,25 +1,80 @@
...
-      {/*  <ListItem*/}
-      {/*    title='JohnDoe'*/}
-      {/*    imgURL='https://avatars.dicebear.com/api/open-peeps/737373.svg'*/}
-      {/*    number={1}*/}
-      {/*    rightText='64 wpm'*/}
-      {/*  />*/}
-      {/*</div>*/}
-      <Button onClick={(): void => startRace()}>Start Race</Button>
-    </div>
+    <Provider store={store}>
```
Comment (supunTE) [inline apps/client/src/pages/room/Room.page.tsx:36]: This `room.page.tsx` contain some test codes which were removed and updates #95

File: apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    if (scrollPosition <= 0) {
+      setBottomCoverVisible(false);
+      setHasOverflow(false);
+    } else {
+      setBottomCoverVisible(true);
+      setHasOverflow(true);
+    }
+
+    containerRef.current?.addEventListener('scroll', containerScroll);
```
Comment (dalanad) [inline apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx]: remove the event listener on cleanup

File: apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  }, [containerRef, playerIds]);
+
+  if (!playerIds?.length || playerIds.length === 0) {
+    return <div></div>;
```
Comment (dalanad) [inline apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx]: is `div` needed or can we use `null`

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx (hunk 2 above)
Comment (supunTE) [reply apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx]: Fixed

File: apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx (hunk 3 above)
Comment (supunTE) [reply apps/client/src/pages/room/templates/player-list/PlayerList.template.tsx]: Fixed

**Review by dalanad — APPROVED** [review]

**Pull request merged**